### PR TITLE
add dashboard api endpoint

### PR DESCRIPTION
**Pull request by sancyx**

### CODE REVIEW

File: pkg/dashboard/dashboard.go [hunk 1]
```diff
@@ -0,0 +1,57 @@
+package dashboard
+
+type Allocatable struct {
```
Comment (sagikazarmark) [inline pkg/dashboard/dashboard.go]: If not reused anywhere else, these structs should go to the api package.

File: main.go [hunk 2]
```diff
@@ -187,6 +187,13 @@ func main() {
 	auth.Install(router)
 
 	basePath := viper.GetString("pipeline.basepath")
+
+	dashboard := router.Group(basePath + "/dashboard/orgs")
+	dashboard.Use(auth.Handler)
+	dashboard.Use(auth.NewAuthorizer(casbinDSN))
```
Comment (bonifaido) [inline main.go]: You can extract and reuse this `Authorizer`.

File: main.go [hunk 3]
```diff
@@ -183,11 +183,20 @@ func main() {
...
+	dashboard.Use(authoriozer)
+	dashboard.Use(api.OrganizationMiddleware)
+	dashboard.GET("/:orgid/clusters", api.GetDashboard)
+
 	v1 := router.Group(basePath + "/api/v1/")
 	v1.GET("/functions", api.ListFunctions)
 	{
 		v1.Use(auth.Handler)
-		v1.Use(auth.NewAuthorizer(casbinDSN))
+		v1.Use(authoriozer)
```
Comment (sagikazarmark) [inline main.go]: Typo: `authoriozer`

File: main.go [hunk 4]
```diff
@@ -183,11 +183,20 @@ func main() {
 	auth.Install(router)
 
 	basePath := viper.GetString("pipeline.basepath")
+
+	authoriozer := auth.NewAuthorizer(casbinDSN)
+
+	dashboard := router.Group(basePath + "/dashboard/orgs")
+	dashboard.Use(auth.Handler)
+	dashboard.Use(authoriozer)
```
Comment (sagikazarmark) [inline main.go]: Typo: `authoriozer`

File: main.go [hunk 5]
```diff
@@ -183,11 +183,20 @@ func main() {
 	auth.Install(router)
 
 	basePath := viper.GetString("pipeline.basepath")
+
+	authoriozer := auth.NewAuthorizer(casbinDSN)
```
Comment (sagikazarmark) [inline main.go]: Typo: `authoriozer`

File: main.go [hunk 6]
```diff
@@ -183,11 +183,20 @@ func main() {
...
+
+	dashboard := router.Group(basePath + "/dashboard/orgs")
```
Comment (sagikazarmark) [inline main.go]: using `path.Join` or some similar solution might be a better choice here

File: main.go [hunk 7]
```diff
@@ -201,6 +210,7 @@ func main() {
 
 			orgs.POST("/:orgid/clusters", api.CreateClusterRequest)
 			//v1.GET("/status", api.Status)
+			orgs.GET("/:orgid/dashboard/clusters", api.GetDashboard)
```
Comment (sagikazarmark) [inline main.go]: Why is it added here?

**Pull request merged**